report/log-splitter.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func main() {
```

Comment:
Let's drop that script. The next evalation results should split right away. Automate all the things!